portability-api/src/main/java/org/dataportabilityproject/webapp/PortabilityApiMain.java

```diff
@@ -15,80 +15,14 @@
  */
 package org.dataportabilityproject.webapp;
 
-import static org.dataportabilityproject.webapp.SetupHandler.Mode.COPY;
-import static org.dataportabilityproject.webapp.SetupHandler.Mode.IMPORT;
-
-import com.sun.net.httpserver.HttpServer;
-import java.net.InetSocketAddress;
-import java.util.concurrent.Executors;
-import org.dataportabilityproject.PortabilityFlags;
-import org.dataportabilityproject.ServiceProviderRegistry;
-import org.dataportabilityproject.cloud.CloudFactoryFactory;
-import org.dataportabilityproject.cloud.interfaces.CloudFactory;
-import org.dataportabilityproject.job.Crypter;
-import org.dataportabilityproject.job.JWTTokenManager;
-import org.dataportabilityproject.job.JobDao;
-import org.dataportabilityproject.job.PortabilityJobFactory;
-import org.dataportabilityproject.job.TokenManager;
-import org.dataportabilityproject.job.UUIDProvider;
+import com.google.inject.Guice;
+import com.google.inject.Injector;
 
 public class PortabilityApiMain {
```

Comment:
Are these your changes to this file? Maybe you need to checkout or revert to the origin/master

Reply:
Hm I think this might be from looking at  a diff before I had merged willard's change. In the UI there should be a button to view diffs from all commits?